.gitignore

```diff
@@ -22,6 +22,9 @@ cmake-build-release
 
 .rosinstall.bak
 
+# 3rd Party Libraries
+external_libs
```

Comment:
if this works but the below one doesn't (line 30), we should get rid of the old one